modules/cloud-sql/variables.tf

```diff
@@ -69,12 +69,6 @@ variable "authorized_networks" {
   # ]
 }
 
-variable "authorized_gae_applications" {
-  description = "A list of Google App Engine (GAE) project names that are allowed to access this instance."
-  type        = list(string)
-  default     = []
-}
-
```

Comment:
Make sure the release notes capture these changes and have a migration guide!

Reply:
Yep don't worry I have a guide prepared!